docs/source/api_reference/index.rst

```diff
@@ -1,7 +1,7 @@
 API Reference
 =============
 
-There are two types of functions useful when working with the pseudopeople
+There are two types of functions useful when working with the ``pseudopeople``
```

Comment:
There are two type of functions available in ``pseudopeople``. ?

Reply:
I think this is referrring to the generate vs noise functions?

Reply:
Hmm, my thought was two over-arching _types_, ie configuration or generation. I'll think a bit

Reply:
"Available" > "useful" imho